packages/cms-cli/commands/create.js

```diff
@@ -39,6 +40,11 @@ const ASSET_PATHS = {
   ),
 };
 
+const PROJECT_REPOSITORIES = {
+  [TYPES['react-app']]: 'cms-react-boilerplate',
```

Comment:
Discussed briefly with @levinkeo but we may want to use a different repo for the react module

Reply:
Would it make sense to have a mono repo? I'm wondering if down the road we have `hs create <insert framework of choice here>-app` it may be easier to manage them and have consistency when referencing things from other projects(like here for instance).

Reply:
Good idea, that's definitely something we're going to have to consider because we'll definitely have at least a few flavors of this.